contracts/truefi2/LoanFactory2.sol

```diff
@@ -172,7 +175,7 @@ contract LoanFactory2 is ILoanFactory2, Initializable {
         uint256 _amount,
         uint256 _term,
         uint256 _apy
-    ) external override onlyFTLA returns (ILoanToken2) {
+    ) external override onlyFTLAOrLoan returns (ILoanToken2) {
```

Comment:
I think this should still be `onlyFTLA`, since only FixedTermLoanAgency should be able to create loan tokens. (It doesn't make sense for loan tokens to create other loan tokens.) I believe you meant to guard `createDebtToken()` behind `onlyTCAOrLoan`?